exec-agent/assembly/assembly.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+
+    Copyright (c) 2012-2016 Codenvy, S.A.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
+    <id>${item}</id>
+    <includeBaseDirectory>true</includeBaseDirectory>
+    <baseDirectory>terminal</baseDirectory>
```

Comment:
Maybe, i haven't changed it yet because:
- executable name should be also changed
- agent name should be also changed

i think that the most suitable name is `exec-agent` but it is another, not really small bunch of work which i think should be done separately